G1ANT.Addon.Net/G1ANT.Addon.Net/NetAddon.cs

```diff
@@ -42,7 +42,7 @@ private void UnpackDrivers()
                 { "VNC.exe", Resources.VNC },
                 { "telnet.exe", Resources.telnet },
             };
-            foreach (var exe in driversDictionary.Where(e => !DoesFileExist(unpackfolder, e.Key) || DoesFileSameLenght(e.Value.Length, unpackfolder, e.Key)))
+            foreach (var exe in driversDictionary.Where(e => !DoesFileExist(unpackfolder, e.Key) || AreFilesOfTheSameLength(e.Value.Length, unpackfolder, e.Key)))
```

Comment:
Same issues as in the other PR - probably missing negation before `AreFilesOfTheSameLength` + bad (unrelated) names for `exe` and `driversDictionary`.